### PR TITLE
New package OutdoorLights

### PR DESCRIPTION
**Pull request by kuzha**
@mwetter:

The current implementation is: outdoor lights is controlled only by the sunUp information. I am not sure if other features need to be added.
The testing is also simple since there is no input for the block, with only three parameters like the SunRiseSet block.

Please review and comment.

### CODE REVIEW

File: Buildings/Controls/OBC/OutdoorLights/DaylightControlled.mo [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+protected
+  CDL.Utilities.SunRiseSet sunRiseSet(
+  final lat=lat,
+  final lon=lon,
+  final timZon=timZon)
+   annotation (Placement(transformation(extent={{-20,-10},{0,10}})));
+
+equation
+  y = if sunRiseSet.sunUp then 0 else 1;
```
Comment (mwetter) [inline Buildings/Controls/OBC/OutdoorLights/DaylightControlled.mo]: While this is valid Modelica, it is not valid in OpenBuildingControl. See http://obc.lbl.gov/specification/cdl.html#equations

The CDL translator modelica-json won't translate such assignments.

Comment (kuzha) [reply Buildings/Controls/OBC/OutdoorLights/DaylightControlled.mo]: CDL blocks are used for the implementation. Please review.

File: Buildings/Controls/OBC/OutdoorLights/DaylightControlled.mo [hunk 2]
```diff
@@ -12,10 +12,19 @@ protected
   final lat=lat,
   final lon=lon,
   final timZon=timZon)
-   annotation (Placement(transformation(extent={{-20,-10},{0,10}})));
+   annotation (Placement(transformation(extent={{-80,-10},{-60,10}})));
+  CDL.Logical.Not not1
```
Comment (mwetter) [inline Buildings/Controls/OBC/OutdoorLights/DaylightControlled.mo]: Every instance needs a short comment. Please update the file accordingly.

Comment (kuzha) [reply Buildings/Controls/OBC/OutdoorLights/DaylightControlled.mo]: Sorry I forgot again...Added already.

**Pull request merged**